inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pxd

```diff
@@ -42,6 +42,7 @@ cdef class InferRequest:
 
 cdef class IENetwork:
     cdef C.IENetwork impl
+    cdef shared_ptr[CExecutableNetwork] _ptr_plugin
```

Comment:
Do we need to create it as shared ptr? I except creation on stack would also work